### ISSUE TITLE
error: A hook (`orm`) failed to load!

### ISSUE THREAD

**Issue by nguyenhuutinh** [jledentu/sails-hook-slugs#2]
is this library support Sails 1?

I tried but got this error:
 slug: {
       type: 'slug',
      from: 'title',
      blacklist: ['search']
    },
```

error: A hook (`orm`) failed to load!
error: Could not tear down the ORM hook.  Error details: Error: Consistency violation: Attempting to tear down a datastore (`default`) which is not currently registered with this adapter.  This is usually due to a race condition in userland code (e.g. attempting to tear down the same ORM instance more than once), or it could be due to a bug in this adapter.  (If you get stumped, reach out at http://sailsjs.com/support.)
```

**Comment by jledentu** [comment]
Hi,
Mmh, the hook actually fails with Sails 1. Sails doesn't accept any more extra properties in model attributes. I'm working on a new version to fix that.

**Comment by jledentu** [comment]
I published a *4.0.0* version that makes this library compatible with Sails 1. Thanks for having reported this issue.
